services/web/server/src/simcore_service_webserver/projects/_nodes_api.py

```diff
@@ -96,10 +95,10 @@ class NodeScreenshot(BaseModel):
     mimetype: str | None = Field(
         default=None,
         description="File's media type or None if unknown. SEE https://www.iana.org/assignments/media-types/media-types.xhtml",
-        example="image/jpeg",
+        examples=["image/jpeg"],
```

Comment:
THOUGTH: we should extend the auto `tests__model__examples` to check these as well. For the moment we only check that the examples in the `model_config` validate against the pydantic class.